Src/WitsmlExplorer.Frontend/services/logObjectService.tsx

```diff
@@ -1,36 +1,70 @@
 import { ErrorDetails } from "models/errorDetails";
-import { LogData } from "models/logData";
+import { LogData, LogDataRequestQuery } from "../models/logData";
 import { ApiClient } from "services/apiClient";
 import AuthorizationService from "services/authorizationService";
 import NotificationService from "services/notificationService";
+import LogCurveInfo from "../models/logCurveInfo";
 
 export default class LogObjectService {
   public static async getLogData(
     wellUid: string,
     wellboreUid: string,
-    logUid: string,
-    mnemonics: string[],
+    requestData: LogDataRequestQuery[],
     startIndexIsInclusive: boolean,
     startIndex: string,
     endIndex: string,
     loadAllData: boolean,
     abortSignal: AbortSignal
   ): Promise<LogData> {
-    if (mnemonics.length === 0) return;
+    if (requestData.length === 0) return;
     const params = [
+      `startIndex=${encodeURIComponent(startIndex)}`,
+      `endIndex=${encodeURIComponent(endIndex)}`,
       `startIndexIsInclusive=${startIndexIsInclusive}`,
       `loadAllData=${loadAllData}`
     ];
-    if (startIndex) params.push(`startIndex=${encodeURIComponent(startIndex)}`);
-    if (endIndex) params.push(`endIndex=${encodeURIComponent(endIndex)}`);
-    const pathName = `/api/wells/${encodeURIComponent(
-      wellUid
-    )}/wellbores/${encodeURIComponent(wellboreUid)}/logs/${encodeURIComponent(
-      logUid
-    )}/logdata?${params.join("&")}`;
+    const pathName = `/api/wells/${wellUid}/wellbores/${wellboreUid}/multilogdata?${params.join(
```

Comment:
We also need the encoding of the well and wellbore uids.

Reply:
fixed